overlord/ifacestate/handlers.go

```diff
@@ -87,7 +87,6 @@ func (m *InterfaceManager) doSetupProfiles(task *state.Task, _ *tomb.Tomb) error
 	// - restore connections based on what is kept in the state
 	//   - if a connection cannot be restored then remove it from the state
 	// - setup the security of all the affected snaps
-	blacklist := m.repo.AutoConnectBlacklist(snapName)
```

Comment:
Should we add a FIXME comment here that we will be adding this back at a future date?